templatefunc.go

```diff
@@ -352,11 +358,25 @@ func parseFormToStruct(form url.Values, objT reflect.Type, objV reflect.Value) e
 		case reflect.Struct:
 			switch fieldT.Type.String() {
 			case "time.Time":
-				format := time.RFC3339
-				if len(tags) > 1 {
-					format = tags[1]
+				var (
+					t   time.Time
+					err error
+				)
+				if len(value) >= 19 {
+					value = value[:19]
+					t, err = time.ParseInLocation(formatDateTime, value, time.Local)
```

Comment:
I don't catch your point much, are you saying the line you mentioned are duplicated?  It's handling three different time format, not dup. And are you meaning to include time format `2006-01-02T15:04:05Z07:00` as well?